### ISSUE TITLE
Manual queue of Vet Center editor notifications email

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#13045]
## Background

The first time the notifications email will be sent out to Vet Center editors, this will be done manually. 

## Acceptance Criteria

- [ ] Notifications email is sent manually
- [ ] A Vet Center equivalent of VAMC drush php-eval "print_r(\Drupal::service('va_gov_notifications.outdated_content')->checkForOutdatedVamcContent());" is created
- [ ] Existing tasks are modified if needed or a new ticket is created if there's no test coverage

### Team
Please check the team(s) that will do this work.

- [ ] `CMS Team`
- [ ] `Public Websites`
- [x] `Facilities`
- [ ] `User support`

**Comment by productmike** [comment]
This is currently blocked -- waiting on Facilities team to massage the need for this release. More than likely will come in 3Q.

**Comment by xiongjaneg** [comment]
per Mike and Jane discussion, Facilities to take first pass due to existing experience with this and then connect with CMS team engineer for knowledge transfer

**Comment by davidmpickett** [comment]
@swirtSJW @omahane - looks like the Drush command may already exist? https://github.com/department-of-veterans-affairs/va.gov-cms/pull/13573

**Comment by davidmpickett** [comment]
Based on discussion in Sprint Planning today, I believe that this ticket is for actually sending the email once it's all implemented and ready to go. I just created a [quick stub](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14816) to list as a blocker.

**Comment by xiongjaneg** [comment]
an unknown is if there are tests for VAMC that can apply or altered to apply with Vet Centers; if none, will need a new ticket to create tests

**Comment by mmiddaugh** [comment]
Email content added to #14816

**Comment by xiongjaneg** [comment]
May not be needed per [Slack thread] (https://dsva.slack.com/archives/C0FQSS30V/p1695332629132279)

**Comment by swirtSJW** [comment]
Closing this as it is not needed.  The first bulk email to Vet Center editors will go out automatically on Oct 1 around 3:30 AM in the same process that sends the VAMC email.
